src/inmanta/server/services/databaseservice.py

```diff
@@ -124,22 +138,26 @@ def _add_gauge(self, name: str, the_gauge: CallbackGauge) -> None:
     def start_monitor(self) -> None:
         """Attach to monitoring system"""
 
+        # TODO not all these are correct / equivalent to previous asyncpg pool implementation
         self._add_gauge(
             "db.connected",
-            CallbackGauge(
-                callback=lambda: 1 if (self._pool is not None and not self._pool._closing and not self._pool._closed) else 0
-            ),
+            CallbackGauge(callback=lambda: 1 if (get_pool() is not None) else 0),
```

Comment:
Does sqlalchemy have a beter pool status than asyncpg? Could be interesting?

Reply:
I don't know if it is better per se, but it does have the following metrics at least (see slack discussion https://inmanta.slack.com/archives/CKRF0C8R3/p1739875570291499?thread_ts=1739871251.554869&cid=CKRF0C8R3)

I'm not sure how to proceed for the db monitoring.
I created [this ticket](https://github.com/inmanta/inmanta-core/issues/8775) to handle this part on its own but maybe it's better to do it now in one go.